Formula/edirect.rb

```diff
@@ -25,10 +24,12 @@ def install
 
     ENV.prepend_create_path "PERL5LIB", prefix/"perl5/lib/perl5"
     ENV["OPENSSL_PREFIX"] = Formula["openssl"].opt_prefix # for Net::SSLeay
-    pms = %w[Encode::Locale File::Listing HTML::Parser HTML::Tagset HTML::Entities HTML::Tree
+    pms = %w[XML::Simple Encode::Locale File::Listing HTML::Parser HTML::Tagset HTML::Entities HTML::Tree
```

Comment:
Does it build without this change?

Reply:
In this case, removing `XML::Simple` caused the build to fail with the following error message:
```
==> brew test brewsci/bio/edirect --verbose
==> FAILED
Testing brewsci/bio/edirect
==> /home/linuxbrew/.linuxbrew/Cellar/edirect/11.6/bin/efetch -version
Can't locate XML/Simple.pm in @INC (you may need to install the XML::Simple module) (@INC contains: /home/linuxbrew/.linuxbrew/Cellar/edirect/11.6/libexec/aux/lib/perl5 /home/linuxbrew/.linuxbrew/Cellar/edirect/11.6/perl5/lib/perl5/x86_64-linux-thread-multi /home/linuxbrew/.linuxbrew/Cellar/edirect/11.6/perl5/lib/perl5 /home/linuxbrew/.linuxbrew/Cellar/perl/5.30.0/lib/perl5/site_perl/5.30.0/x86_64-linux-thread-multi /home/linuxbrew/.linuxbrew/Cellar/perl/5.30.0/lib/perl5/site_perl/5.30.0 /home/linuxbrew/.linuxbrew/Cellar/perl/5.30.0/lib/perl5/5.30.0/x86_64-linux-thread-multi /home/linuxbrew/.linuxbrew/Cellar/perl/5.30.0/lib/perl5/5.30.0 /home/linuxbrew/.linuxbrew/lib/perl5/site_perl/5.30.0/x86_64-linux-thread-multi /home/linuxbrew/.linuxbrew/lib/perl5/site_perl/5.30.0) at /home/linuxbrew/.linuxbrew/Cellar/edirect/11.6/libexec/edirect.pl line 79.
BEGIN failed--compilation aborted at /home/linuxbrew/.linuxbrew/Cellar/edirect/11.6/libexec/edirect.pl line 79.
Error: brewsci/bio/edirect: failed
An exception occurred within a child process:
  Test::Unit::AssertionFailedError: <0> expected but was
<2>.
```

Reply:
Great. Please add `XML::Simple` back in.